Riot/Modules/Room/Location/RoomTimelineLocationView.swift

```diff
@@ -104,7 +104,26 @@ class RoomTimelineLocationView: UIView, NibLoadable, Themable, MGLMapViewDelegat
     @IBOutlet private var rightButton: UIButton!
     @IBOutlet private var activityIndicatorView: UIActivityIndicatorView!
     
+    @IBOutlet private var mapLoadingErrorContainerView: UIView!
+    @IBOutlet private var mapLoadingErrorImageView: UIImageView!
+    @IBOutlet private var mapLoadingErrorMessageLabel: UILabel!
+    
     private var mapView: MGLMapView!
+    
+    private var _isMapViewLoadingFailed: Bool = false
+    
+    private var isMapViewLoadingFailed: Bool {
```

Comment:
Using an "internal" property `_isMapViewLoadingFailed ` seems odd to me. I believe you can achieve it without `_isMapViewLoadingFailed` and with the following code

```
   private var isMapViewLoadingFailed: Bool {
        didSet {
             if oldValue != isMapViewLoadingFailed {
                 self.mapViewLoadingStateDidChange()
             }
         }
     }
```

Reply:
Indeed lot more cleaner